src/App.css

```diff
@@ -17,6 +17,20 @@ body {
   position: relative;
 }
 
```

Comment:
| Zeplin's Design | @kifle23 Design |
| --- | --- |
| ![image](https://github.com/kifle23/Bookstore/assets/93039359/e630a8ee-6ee3-47e3-8299-a73112b8e273) | ![image](https://github.com/kifle23/Bookstore/assets/93039359/3fc4bd3c-38d1-4d06-b786-128838c37247) |

- [ ] It would be nice if you follow the project requirements and make your body background color as same as Zeplin's design 👍 to make your project professional 😊


🤔 Hint: Right now you are using `#ededed` for your body background color but you should use `#fafafa`